tests/testthat/test-Source_datasets.R

```diff
@@ -117,12 +121,19 @@ names_EMP_Micro<-readr::read_csv(file.path(Data_folder, EMP_Micro_file),
 
 # FRP Macro ---------------------------------------------------------------
 
-Tryer(n=3, fun=download.file, url="https://pasta.lternet.edu/package/data/eml/edi/269/2/630f16b33a9cbf75f1989fc18690a6b3",
-           destfile=file.path(Data_folder, "bugsFRP2018.csv"), mode="wb", method="curl")
 
-names_FRP_Macro <- readr::read_csv(file.path(Data_folder, "bugsFRP2018.csv"),
-                                   col_types = cols(.default=col_character()))%>%
-  names()
+Tryer(n=3, fun=utils::download.file, url="https://portal.edirepository.org/nis/dataviewer?packageid=edi.269.3&entityid=8de785ee47220f3893654478c79b5f8f",
+      destfile=file.path(Data_folder, "bugsFRP2021.csv"), mode="wb", method=Download_method)
+Tryer(n=3, fun=utils::download.file, url="https://portal.edirepository.org/nis/dataviewer?packageid=edi.269.3&entityid=fa84750e51c319d309f97357b7d34315",
+      destfile=file.path(Data_folder, "sitesFRP2021.csv"), mode="wb", method=Download_method)
+
+zoo_FRP_Macro <- readr::read_csv(file.path(Data_folder, "bugsFRP2021.csv"), na=c("", "NA"))
+sites_FRP_Macro <- readr::read_csv(file.path(Data_folder, "sitesFRP2021.csv"), na=c("", "NA"))
+
+#join environmental data to taxa counts and fix some wonky names
+FRP_allmac = dplyr::left_join(dplyr::select(zoo_FRP_Macro, -Date, -Location), sites_FRP_Macro, by = "VisitNo")
```

Comment:
For meso and macro, instead of joining them to the site data and then checking names, it would be better to just check the column names separately for each table, using the raw column names from EDI

Reply:
Done!